lib/i18n/tasks/scanners/results/occurrence.rb

```diff
@@ -48,7 +48,7 @@ def initialize(path:, pos:, line_num:, line_pos:, line:, raw_key: nil, default_a
         # rubocop:enable Metrics/ParameterLists
 
         def inspect
-          "Occurrence(#{@path}:#{@line_num}:#{@line_pos}:#{@pos}:#{@raw_key}:#{@default_arg}:#{@line})"
```

Comment:
@glebm Not sure what you think about this, can revert it 🙂

Reply:
`"#{@path}:#{@line_num}"` is standard notation, let's remove the field names there but keep the rest

Reply:
Going to merge this for now and release 1.0.2 anyway